scripts/test/Muon/fitting_tab_widget/fitting_tab_presenter_test.py

```diff
@@ -39,6 +41,62 @@ def test_handle_select_fit_data_clicked_updates_current_run_list(self, dialog_mo
                           'MUSR62260; Pair Asym; long; #1', 'MUSR62260; PhaseQuad; PhaseTable MUSR62260',
                           'MUSR62260; PhaseQuad; PhaseTable MUSR62261'])
 
+    def test_that_changeing_fitting_type_to_multiple_fit_changes_workspace_selector_combo_label(
+            self):
+        self.view.sequential_fit_radio.toggle()
+
+        self.assertEqual(self.view.workspace_combo_box_label.text(), 'Display parameters for')
+
+    def test_that_changeing_fit_type_to_single_fit_updates_label(self):
+        self.view.sequential_fit_radio.toggle()
+        self.view.single_fit_radio.toggle()
+
+        self.assertEqual(self.view.workspace_combo_box_label.text(), 'Select Workspace')
+
+    def test_when_fit_is_clicked_in_single_mode_the_fit_function_string_workspace_and_additional_options_are_passed_to_fit(
+            self):
+        self.presenter.selected_data = ['Input Workspace Name']
+        self.view.function_browser.setFunction('name=GausOsc,A=0.2,Sigma=0.2,Frequency=0.1,Phi=0')
+        self.view.fit_button.clicked.emit(True)
+
+        self.presenter.model.do_single_fit.assert_called_once_with(
+            {'Function': mock.ANY, 'InputWorkspace': 'Input Workspace Name',
```

Comment:
whats this `mock.ANY`?

Reply:
It's a comparitor which always compares true, see https://docs.python.org/3/library/unittest.mock.html

In this case it is used because the function can not be easily compared with because it is equal to FunctionObject created internally we don't have any access to. It would be better if there were some separate tests of the function below however so I'll add them.